content/en/docs/Getting Started/pipelines.md

```diff
@@ -81,16 +88,23 @@ both the *Hello World!* and *Goodbye World!* Tasks.
             - hello
           taskRef:
             name: goodbye
+          params:
+          - name: username
+            value: $(params.username)
     ```
 
-1.  Apply your Pipeline configuration to your cluster:
+    The Pipeline defines the parameter `username`, that parameter is then passed to
+    the  `goodbye` Task.
+
+1.  Apply the Pipeline configuration to your cluster:
 
     ```bash
     kubectl apply --filename hello-goodbye-pipeline.yaml
     ```
 
-1.  Instantiate your Pipeline with a `PipelineRun` object. Create a new file
-    named `hello-goodbye-pipeline-run.yaml` with the following content:
+1.  A **Pipeline Run**, represented in the API as an object of kind
+    `PipelineRun`, instantiates a Pipeline. To create  Pipeline Run, create a
```

Comment:
Has there been a discussion about using camel case in regular text? It makes sense when referring to a field or an API object, but when talking about the concept itself, saying "pipeline run" seems like a totally reasonable noun to me. Same with "trigger template", "trigger bind", "task run" and so on.

I'm going to gather some examples from the space and start a discussion about this on Slack, it would be nice to have consistent "Branding Guidelines" for Tekton and first-level Tekton objects.

Reply:
Ok, K8s uses [camel case with sentence capitalization](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects). I'm going to follow that convention.